charts/nextcloud/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+This Helm-Chart use semantic-releases, so only major version contains breaking changes.
```

Comment:
Or when a new major Nextcloud version is used.
It's not a breaking change because it only changes the default, but we have learned that it causes trouble for many users, so we bump the major version anyway.

Reply:
Yes